.github/workflows/ci.yml

```diff
@@ -91,8 +91,8 @@ jobs:
         run: |
           rustup set profile minimal
           # Pin to older version until a clippy regression is fixed
```

Comment:
This comment seems to imply that it's desired to switch back to `rustup update stable` - which from git log history appears to be how it used to work.

Do you want to use
```
rustup update stable
rustup default stable
```


Alternately, remove this comment about pinning an older version?